Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionPublishToQCOperations.cs

```diff
@@ -82,5 +119,57 @@ public void Execute(IReportInfo RI)
             }
         }
 
+        private BusinessFlow ConvertRunSetToBF(IReportInfo reportInfo)
+        {
+            RunsetExecutor runSetExec = WorkSpace.Instance.RunsetExecutor;
+            try
+            {
+                if (reportInfo == null)
+                {
+                    return null;
+                }
+
+                //Create Business Flow
+                BusinessFlow virtualBF = new BusinessFlow();
+                virtualBF.Name = runSetExec.RunSetConfig.Name;
+                virtualBF.Description = runSetExec.RunSetConfig.Description;
+                virtualBF.Status = BusinessFlow.eBusinessFlowStatus.Development;
+                virtualBF.RunStatus = runSetExec.RunSetConfig.RunSetExecutionStatus;
+                virtualBF.Activities = new ObservableList<Activity>();
+                foreach (GingerRunner runSetrunner in runSetExec.Runners)
+                {
+                    // if executor is null when run if from file
+                    if(runSetrunner.Executor is null)
+                    {
+                        runSetrunner.Executor = new GingerExecutionEngine(runSetrunner);
```

Comment:
what is this 'Executor'? you need to loop over Runner.BusinessFlows list

Reply:
arvind changes